### PR TITLE
Remove expired content from the cache

### PR DESCRIPTION
**Pull request by piuccio**
Should free some memory

### CODE REVIEW

File: facia-tool/public/js/modules/cache.js [hunk 1]
```diff
@@ -41,6 +41,7 @@ function get(pot, key) {
     if (!obj) { return; }
 
     if (+new overrides.Date() - obj.time > expiry) {
+        p[key] = null;
```
Comment (rich-nguyen) [inline facia-tool/public/js/modules/cache.js]: is this the same as `delete`?

Comment (piuccio) [inline facia-tool/public/js/modules/cache.js]: true, never been a big fan of `delete`

**Comment by piuccio** [comment]
@rich-nguyen can I get a +1 now? please :smile:

**Comment by rich-nguyen** [comment]
:+1: !

**Pull request merged**